api/server.js

```diff
@@ -144,7 +144,7 @@ app.get("/bookshelves/get/:username/:bookshelf", async function(req, res) {
 
 			let info = [username, bookshelf];
 			let result = await getBook(info);
-			if (result.length < 1) {
+			if (!result) {
```

Comment:
this may or may not work depending on what the mysql module returns in both of these situations. We need to test this post merge